webdriver-spec.html

```diff
@@ -1888,16 +1888,17 @@
 </table>
 
 <p>The <dfn>Get Current Url</dfn> <a>command</a>
- returns the URL of the <a>current top-level browsing context</a>.
+ returns the URL of the <a>current browsing context</a>.
```

Comment:
Did you see http://lists.w3.org/Archives/Public/public-browser-tools-testing/2015JulSep/0035.html?